### PR TITLE
Css introduction

### PR DESCRIPTION
**Pull request by Stephs15**

### CODE REVIEW

File: src/lessons/03_introduction_to_css/introduction_to_css.slides.html [hunk 1]
```diff
@@ -0,0 +1,620 @@
...
+              <li class="fragment">The homework!</li>
+            </ul>
+          </section>
+        </section>
+
+        <section>
+          <!--SLIDE 3.1-->
+          <section data-auto-animate data-timing="10">
+            <h2>What is CSS?</h2>
+            <h3 class="fragment">Cascade Style Sheet</h3>
```
Comment (oktupol) [inline src/lessons/03_introduction_to_css/introduction_to_css.slides.html]: Minor correction
```suggestion
            <h3 class="fragment">Cascading Style Sheet</h3>
```

File: src/lessons/03_introduction_to_css/introduction_to_css.slides.html [hunk 2]
```diff
@@ -0,0 +1,620 @@
...
+            <br />
+            <p class="fragment">External</p>
+          </section>
+          <!--SLIDE 4.2-->
+          <section>
+            <h2>Inline CSS</h2>
+            <pre>
+              <code>
+                <script type="text/template">
+                  <p style="color:red">I am a red paragraph</p>
```
Comment (oktupol) [inline src/lessons/03_introduction_to_css/introduction_to_css.slides.html:86]: Not sure whether the indentation is intentional, but if not, you can get rid of it by adding `data-trim` into the `code` tag

File: src/style/layout/codeStyle.scss [hunk 3]
```diff
@@ -0,0 +1,4 @@
+code {
+  border-radius: 10px;
+  border: 5px solid #4a90e2;
+}
```
Comment (oktupol) [inline src/style/layout/codeStyle.scss]: To be honest, I am not a big fan of this. It looks fine on the slides that have only one code example, but those with multiple are getting quite busy

File: src/lessons/03_introduction_to_css/introduction_to_css.slides.html [hunk 4]
```diff
@@ -0,0 +1,620 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>Introduction to CSS</title>
+    <meta name="date" content="2021-03-18" />
+    <!-- TODO: ADD content -->
+    <meta name="description" content="" />
```
Comment (oktupol) [inline src/lessons/03_introduction_to_css/introduction_to_css.slides.html]: 👀

File: src/style/layout/codeStyle.scss (hunk 3 above)
Comment (oktupol) [reply src/style/layout/codeStyle.scss]: Oh, and it breaks the existing slides.

I'd find it okay if you tie it to a class

File: src/lessons/03_introduction_to_css/introduction_to_css.slides.html [hunk 5]
```diff
@@ -0,0 +1,620 @@
...
+            <h2>Combinators</h2>
+            <div class="r-stack">
+              <div class="fragment fade-out" data-fragment-index="0">
+                <h5>Descendant</h5>
+                <h6>Selects all elements that are inside another</h6>
+                <div class="columns align-center justify-center">
+                  <div>
+                    <pre>
+                      <code data-trim>
+                          /* Selects the last p of the parent  */
```
Comment (oktupol) [inline src/lessons/03_introduction_to_css/introduction_to_css.slides.html]: This is wrong 😁 
This comment appears a few more times down below

File: src/lessons/03_introduction_to_css/introduction_to_css.slides.html [hunk 6]
```diff
@@ -0,0 +1,620 @@
...
+                      </code>
+                    </pre>
+                  </div>
+                  <div style="min-width: 500px">
+                    <pre>
+                      <code class="html" data-trim>
+                        <div>Here is some content.</div>
+                        <span>More content.</span>
+                        <p>And here is a red paragraph!</p>
+                        <p>And here is a another red paragraph!</p>
```
Comment (oktupol) [inline src/lessons/03_introduction_to_css/introduction_to_css.slides.html]:
```suggestion
                        <p>And here is a blue paragraph!</p>
                        <p>And here is a another blue paragraph!</p>
```

**Review by oktupol — COMMENTED** [review]
I added a few suggestions 🙂

**Review by guicheffer — COMMENTED** [review]
Apart from @oktupol's comments, looks good to me so far - will dig into testing before the class, but consider it as a green light 👍🏼

File: src/lessons/03_introduction_to_css/introduction_to_css.slides.html (hunk 5 above)
Comment (Stephs15) [reply src/lessons/03_introduction_to_css/introduction_to_css.slides.html]: Oops, copy paste alert 😅

File: src/style/layout/codeStyle.scss (hunk 3 above)
Comment (Stephs15) [reply src/style/layout/codeStyle.scss]: sure!

File: src/lessons/03_introduction_to_css/introduction_to_css.slides.html [hunk 7]
```diff
@@ -0,0 +1,614 @@
...
+              </code>
+            </pre>
+          </section>
+          <section>
+            <h2>Pseudo-elements Selectors</h2>
+            <div class="columns align-center justify-center">
+              <div class="fragment">
+                <pre>
+                  <code class="formattedCode" data-trim>
+                      /* Selects the last p of the parent  */
```
Comment (oktupol) [inline src/lessons/03_introduction_to_css/introduction_to_css.slides.html]: There are still a two of these in wrong places 🙂

**Review by oktupol — APPROVED** [review]
Looks good to me 👍🏽

**Pull request merged**